src/node.ts

```diff
@@ -0,0 +1,396 @@
+// Copyright (c) 2018 Zilliqa
+// This source code is being disclosed to you solely for the purpose of your participation in
+// testing Zilliqa. You may view, compile and run the code for that purpose and pursuant to
+// the protocols and algorithms that are programmed into, and intended by, the code. You may
+// not do anything else with the code without express permission from Zilliqa Research Pte. Ltd.,
+// including modifying or publishing the code (or any part of it), and developing or forming
+// another public or private blockchain network. This source code is provided ‘as is’ and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose
+// and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+import { validateArgs } from './util';
+import * as util from './util';
+
+type callback = (error: any, data: any) => any;
+
+const utils = {
```

Comment:
I think this could be a type/interface instead of a mapping object. 

Another option would be splitting the util file, one will have utils that will be exposed on the interface of the library and other/other that are private.

Reply:
I agree. Unfortunately we have limited time now for this; would you be willing to send a PR? We would be most grateful for the help :)

Reply:
I agree. Unfortunately we have limited time now for this; would you be willing to send a PR? We would be most grateful for the help :)

Reply:
I agree. Unfortunately we have limited time now for this; would you be willing to send a PR? We would be most grateful for the help :)

Reply:
I agree. Unfortunately we have limited time now for this; would you be willing to send a PR? We would be most grateful for the help :)